### DISCUSSION TITLE
Add option to specify config file in self-hosted bot

### DISCUSSION THREAD

**Discussion by moalam-vmw** [renovatebot/renovate#14158] [Request Help]
### What would you like Renovate to be able to do?

Add an option to the self hosted config to allow one of the other config file names to be used in self-hosted config. 

### If you have any ideas on how this should be implemented, please tell us here.

There is already an option for onboardingconfigfilename and a configfilename option would be good too. 

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
Please create discussions first in future

**Comment by rarkins** [comment]
I think you mean this? https://docs.renovatebot.com/getting-started/running/#global-config
